chains/zircuit/addresses.yaml

```diff
@@ -3,7 +3,7 @@ domainRoutingIsm: "0x93e62F2c9a89f8b1e62A3c726Bbf6BD1A7b7FA3e"
 domainRoutingIsmFactory: "0x1052eF3419f26Bec74Ed7CEf4a4FA6812Bc09908"
 fallbackRoutingHook: "0xDa7cECb05C4aeB02c1aFDE277d4306a2da7Bd762"
 interchainGasPaymaster: "0x03cF708E42C89623bd83B281A56935cB562b9258"
-interchainSecurityModule: "0x636bB3fCa3B74B942bA8C258c44109384B252979"
+interchainSecurityModule: "0x817D346Add1D08Af124a90e43123111c9d6813B3"
 mailbox: "0xc2FbB9411186AB3b1a6AFCCA702D1a80B48b197c"
 merkleTreeHook: "0x4C97D35c668EE5194a13c8DE8Afc18cce40C9F28"
 pausableHook: "0x4Eb82Ee35b0a1c1d776E3a3B547f9A9bA6FCC9f2"
```

Comment:
did the pausableIsm (line below) also get updated? thought you said it deployed

Reply:
yeah nice catch - check-deploy doesn't auto-update local registry atm!